examples/node-custom-rpc/src/main.rs

```diff
@@ -16,28 +16,24 @@ use clap::Parser;
 use jsonrpsee::{core::RpcResult, proc_macros::rpc};
 use reth::{chainspec::EthereumChainSpecParser, cli::Cli};
 use reth_node_ethereum::EthereumNode;
-use reth_transaction_pool::TransactionPool;
+use reth_transaction_pool::{TransactionPool, PoolSize};
+use tracing_subscriber;
 
 fn main() {
+    tracing_subscriber::fmt::init(); // Improved logging
+
     Cli::<EthereumChainSpecParser, RethCliTxpoolExt>::parse()
         .run(|builder, args| async move {
             let handle = builder
                 .node(EthereumNode::default())
                 .extend_rpc_modules(move |ctx| {
-                    if !args.enable_ext {
-                        return Ok(())
-                    }
-
-                    // here we get the configured pool.
-                    let pool = ctx.pool().clone();
-
-                    let ext = TxpoolExt { pool };
-
-                    // now we merge our extension namespace into all configured transports
```

Comment:
I want to keep these docs because this is an example and these provide helpful context